packages/ckeditor5-find-and-replace/src/ui/findandreplaceformview.js

```diff
@@ -337,6 +285,162 @@ export default class FindAndReplaceFormView extends View {
 		}, { priority: 'high' } );
 	}
 
+	/**
+	 * Initialization of focusables elements of the form.
+	 *
+	 * @private
```

Comment:
The order of functions in class should be:

*   public
*   protected
*   private

Both \_assignViews and \_initFocusables occurrs before focus (which is a public method).